src/plotvariables.jl

```diff
@@ -95,4 +95,5 @@ function plotvariables(solution, xrange, variables::Vector{<:NTuple{4, Any}}; na
     end
 
     plot(traces, Layout(layout))
+    save && savefig(plot(traces, Layout(layout)), "docs/src/assets/plots/" * name * ".svg")
```

Comment:
Some remarks:
1. it's better to reuse the plot inside the `savefig` call;
2. I think the figure should be saved into the root of the folder; and
3. return the plot explicitly.

For example,
```
    p = plot(traces, Layout(layout))
    save && savefig(p, "./" * name * ".svg")

    return p
```